NOTICE.md

```diff
@@ -124,6 +124,18 @@ natefinch/lumberjack (2.0.0)
 * Project: https://github.com/natefinch/lumberjack
 * Source:  https://github.com/natefinch/lumberjack/releases/tag/v2.0
 
+github.com/caarlos0/env (6.10.1)
+
+* License: MIT License
+* Project: https://github.com/caarlos0/env
+* Source:  https://github.com/caarlos0/env/releases/tag/v6.10.1
+
+github.com/stretchr/testify (1.8.1)
```

Comment:
For the binary components, the notice contains only the libraries that will be included in the binary artifact. The test libraries should not be there.